<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="dateLayout">
        <item>
         <widget class="QLabel" name="dateLabel">
          <property name="text">
           <string>Date (year/month/day):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::YearSection</enum>
          </property>
          <property name="displayFormat">
           <string>yyyy.MM.dd</string>
          </property>
          <property name="calendarPopup">
           <bool>false</bool>
          </property>
          <property name="currentSectionIndex">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="winnersLayout">
        <item>
         <widget class="QLabel" name="winnersLabel">
          <property name="text">
           <string>Winners:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="winnersLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="losersLayout">
        <item>
         <widget class="QLabel" name="losersLabel">
          <property name="text">
           <string>Losers:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="losersLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="amountLayout">
        <item>
         <widget class="QLabel" name="amountLabel">
          <property name="text">
           <string>Amount:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="amountLineEdit">
          <property name="maxLength">
           <number>32767</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="topButtonLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomButtonLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="statisticsLayout">
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="moneyWonLayout">
        <item>
         <widget class="QLabel" name="moneyWonLabel">
          <property name="text">
           <string>Money won</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="moneyWonLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="totalMoneyLayout">
        <item>
         <widget class="QLabel" name="totalMoneyLabel">
          <property name="text">
           <string>Total money</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="totalMoneyLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="betsWonLayout">
        <item>
         <widget class="QLabel" name="betsWonLabel">
          <property name="text">
           <string>Bets won</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="betsWonLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <layout class="QVBoxLayout" name="moneyLostLayout">
        <item>
         <widget class="QLabel" name="moneyLostLabel">
          <property name="text">
           <string>Money lost</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="moneyLostLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="totalBetsLayout">
        <item>
         <widget class="QLabel" name="totalBetsLabel">
          <property name="text">
           <string>Total bets</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="totalBetsLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QVBoxLayout" name="maxWonLayout">
        <item>
         <widget class="QLabel" name="maxWonLabel">
          <property name="text">
           <string>Max money won</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="maxWonLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <layout class="QVBoxLayout" name="maxLostLayout">
        <item>
         <widget class="QLabel" name="maxLostLabel">
          <property name="text">
           <string>Max money lost</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="maxLostLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QVBoxLayout" name="betsLostLayout">
        <item>
         <widget class="QLabel" name="betsLostLabel">
          <property name="text">
           <string>Bets lost</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="betsLostLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <layout class="QVBoxLayout" name="bestTeamWinsLayout">
        <item>
         <widget class="QLabel" name="bestTeamWinsLabel">
          <property name="text">
           <string>Best Team (most wins)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="bestTeamWinsLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QVBoxLayout" name="bestTeamMoneyLayout">
        <item>
         <widget class="QLabel" name="bestTeamMoneyLabel">
          <property name="text">
           <string>Best Team (most money)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="bestTeamMoneyLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="2">
       <layout class="QVBoxLayout" name="worstTeamLossesLayout">
        <item>
         <widget class="QLabel" name="worstTeamLossesBetsLabel">
          <property name="text">
           <string>Worst team (most losses)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="worstTeamLossesLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCustomPlot" name="plot" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>tableView</zorder>
   <zorder></zorder>
   <zorder>plot</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
